src/main/java/dk/cngroup/lentils/controller/TeamController.java

```diff
@@ -27,20 +24,20 @@ public TeamController(TeamService teamService) {
         this.teamService = teamService;
     }
 
-    @GetMapping(value = "/add")
+    @GetMapping(value = "/main")
```

Comment:
Tady bych nechal bufd `@GetMapping(value = "/")` a nebo `@GetMapping(value = "/all")`. Vsechny url mame v podstate ve tvaru @GetMapping(value = "<entity>/<action>") a akce `main` mi nedava smysl. Je to blbost, ale at to mame jednotne trochu..

Reply:
Zmenil jsem mapping na (value = "")